kc-cloudtrust-module/kc-cloudtrust-common/src/main/java/io/cloudtrust/keycloak/FormUtils.java

```diff
@@ -0,0 +1,67 @@
+package io.cloudtrust.keycloak;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.keycloak.authentication.AuthenticationFlowContext;
+import org.keycloak.authentication.AuthenticationSelectionOption;
+import org.keycloak.credential.CredentialProvider;
+import org.keycloak.credential.CredentialTypeMetadata;
+import org.keycloak.credential.CredentialTypeMetadataContext;
+import org.keycloak.forms.login.LoginFormsProvider;
+import org.keycloak.models.AuthenticationExecutionModel;
+
+public class FormUtils {
```

Comment:
I'd love some doc about the goal of this class...

Reply:
Class commented

Reply:
Thanks :-)